overview/saml.mdx

```diff
@@ -15,6 +15,45 @@ which users have access to your product. They could give everyone at the company
 access, specify teams like only the engineering team, or even pick specific
 people to give access.
 
+### SAML Setup Guide
+
+
+
+1. In the Organization Settings for your project, make sure that SAML is enabled and that all orgs can setup SAML.
+
+![organization settings](/images/saml-setup-1.png)
+
+2. Under Users, make sure that there is at least one test user with Admin or Owner priviledges.
```

Comment:
This step isn't necessary yet, because the organization doesn't exist yet (it's created in step 5)